apps/src/p5lab/spritelab/commands/worldCommands.js

```diff
@@ -24,13 +24,18 @@ export const commands = {
 
   getTime(unit) {
     if (unit === 'seconds') {
-      return this.getAdjustedWorldTime() || 0;
+      return this.getAdjustedWorldTime() - this.timerResetTime || 0;
     } else if (unit === 'frames') {
-      return this.p5.World.frameCount || 0;
+      return this.p5.World.frameCount - this.timerResetFrames || 0;
     }
     return 0;
   },
 
+  resetTimer() {
+    this.timerResetTime = this.getAdjustedWorldTime();
+    this.timerResetFrames = this.p5.World.frameCount;
```

Comment:
nit: we should use the helper that aliases this value in case we ever need to adjust it:

https://github.com/code-dot-org/code-dot-org/blob/8ed2288e123f757d765108ebdd1beb450f293cf8/apps/src/p5lab/spritelab/CoreLibrary.js#L45-L47

Reply:
Didn't realize we had this! Updated.